vroom/clangformat.vroom

```diff
@@ -144,7 +144,7 @@ that clang-format has a version >= 3.4).
   :FormatCode clang-format
   ! clang-format-3.3 --version .*
   $ clang-format version 3.3.0 (tags/testing)
-  ! clang-format-3.3 -style file .*2>.*
+  ! clang-format-3.3 -style file -lines 1:1 2>.*
```

Comment:
Not strictly needed for this change, but I changed this assertion because I noticed that the previous pattern would have also allowed -cursor to appear here, even though we're trying to test that it doesn't. (And we need to do something similar below.)